kernel/src/main/scala/cats/Eq.scala

```diff
@@ -59,11 +61,6 @@ trait EqFunctions {
 object Eq extends EqFunctions {
```

Comment:
Right, simulacrum doesn't generate forwarders on the companion object. We should discuss whether or not typeclassic should support this.